examples/pennylane/0_Getting_started/0_Getting_started.ipynb

```diff
@@ -7,17 +7,6 @@
     "# Combining PennyLane with Amazon Braket"
```

Comment:
<blockquote>You can run your local Python code as an Amazon Braket hybrid job by annotating your code with an @hybrid_job decorator,</blockquote><p>You can run your local Python code as an Amazon Braket hybrid job by annotating your code with a <code>@hybrid_job</code> decorator,</p><p> </p><p>Also link to BYOC.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/393/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>